pkg/agent/openflow/pod_connectivity.go

```diff
@@ -168,7 +168,7 @@ func (f *featurePodConnectivity) initFlows() []*openflow15.FlowMod {
 	// Add flow to ensure the liveliness check packet could be forwarded correctly.
 	flows = append(flows, f.localProbeFlows()...)
 
-	if f.networkConfig.TrafficEncapMode.SupportsEncap() {
+	if f.tunnelPort != 0 {
```

Comment:
there is another reference of `t.tunnelPort` in `flowsToTrace` func, we should do the same for it?

Reply:
yes, thanks for catching this